lib/Github/Api/Issue/Comments.php

```diff
@@ -41,18 +41,24 @@ public function configure($bodyType = null)
      *
      * @link https://developer.github.com/v3/issues/comments/#list-comments-on-an-issue
      *
-     * @param string $username
-     * @param string $repository
-     * @param int    $issue
-     * @param int    $page
+     * @param string    $username
+     * @param string    $repository
+     * @param int       $issue
+     * @param int|array $page
```

Comment:
Also add a description to the page parameter, to indicate that it's deprecated to provide an integer here